firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -269,6 +274,29 @@ static percent_t currentEtbDuty;
 
 		etb1.dcMotor.Set(ETB_PERCENT_TO_DUTY(currentEtbDuty));
 
+	   	float etbError = absF(targetPosition - actualThrottlePosition); // Used for I-Term limiter and Error-detection
+
+
+        // ETB Deviation monitoring
+        if(etbError < 3 ){
```

Comment:
I have to say that this starts to looks like real code!

On a more boring note, code formatting is important. Nicer way to write the same would be "if (etbError < 3) {"

Reply:
I should check in my clang-format file, and we can create a gate/ci for it.